shared/js/initApp.js

```diff
@@ -13,7 +13,8 @@ window.apps = {
   load: loadApp,
   // Legacy Blockly initialization that was moved here from _blockly.html.haml.
   // Modifies `appOptions` with some default values in `baseOptions`.
-  setup: function () {
+  // Move blockly-specific setup function out of shared and back into dashboard.
+  setupBlockly: function () {
```

Comment:
While we are here, can anyone ( @joshlory @Bjvanminnen @wjordan  ) explain why this setup code had to move out of _blockly.html.haml ? Is it needed for grunt dev?